### PR TITLE
config: turn on lite-init-stats and force-init-stats by default

### PR DESCRIPTION
**Pull request by xuyifangreeneyes**
### What problem does this PR solve?

Issue Number: ref #42160

Problem Summary:

### What is changed and how it works?

Turn on `lite-init-stats` and `force-init-stats` by default.

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: statistics/handle/updatetest/update_test.go [hunk 1]
```diff
@@ -446,6 +446,8 @@ func TestAutoUpdate(t *testing.T) {
 		tableInfo = tbl.Meta()
 		h.HandleAutoAnalyze(is)
 		require.NoError(t, h.Update(is))
+		testKit.MustExec("explain select * from t where a > 'a'")
+		require.NoError(t, h.LoadNeededHistograms())
```
Comment (xuyifangreeneyes) [inline statistics/handle/updatetest/update_test.go:450]: After turning on `lite-init-stats` by default, we need to use `LoadNeededHistograms` to load index stats when lease is not zero.

File: statistics/integration_test.go [hunk 2]
```diff
@@ -667,11 +667,7 @@ func TestShowHistogramsLoadStatus(t *testing.T) {
 	require.NoError(t, h.Update(dom.InfoSchema()))
 	rows := tk.MustQuery("show stats_histograms where db_name = 'test' and table_name = 't'").Rows()
 	for _, row := range rows {
-		if row[3] == "a" || row[3] == "idx" {
-			require.Equal(t, "allLoaded", row[10].(string))
-		} else {
-			require.Equal(t, "allEvicted", row[10].(string))
-		}
```
Comment (xuyifangreeneyes) [inline statistics/integration_test.go:674]: After turning on `lite-init-stats` by default, neither pk stats nor index stats are loaded into stats cache when lease is not zero.

**Review by hawkingrei — APPROVED** [review]

**Review by hawkingrei — APPROVED** [review]

File: executor/test/analyzetest/analyze_test.go [hunk 3]
```diff
@@ -1077,16 +1077,10 @@ PARTITION BY RANGE ( a ) (
 	tk.MustExec("insert into t values (1,1,1),(2,1,2),(3,1,3),(4,1,4),(5,1,5),(6,1,6),(7,7,7),(8,8,8),(9,9,9),(10,10,10),(11,11,11),(12,12,12),(13,13,13),(14,14,14)")
 
 	h := dom.StatsHandle()
-	oriLease := h.Lease()
-	h.SetLease(1)
-	defer func() {
-		h.SetLease(oriLease)
-	}()
```
[the remark this answers is not in the document]
Comment (xuyifangreeneyes) [reply executor/test/analyzetest/analyze_test.go:1084]: https://github.com/pingcap/tidb/blob/master/testkit/mockstore.go#L82

**Review by time-and-fate — APPROVED** [review]

**Review by qw4990 — APPROVED** [review]

**Pull request merged**